.circleci/config.yml

```diff
@@ -25,7 +25,7 @@ jobs:
           name: Pushing documentation to gh-pages
           command: |
             pip install --user ghp-import
-            ghp-import -n -p docs/_build/html
+            ghp-import --no-jekyll --push --message "Update documentation [skip ci]" docs/_build/html
```

Comment:
I had to lookup what -n and -p meant, so I figure it made more sense to be explicit for a tool like ghp-import which isn't used frequently by everyone.